src/components/screen/Calender/Calender.tsx

```diff
@@ -0,0 +1,87 @@
+import { format, getDate, getDay, addMonths, subMonths,getMonth , endOfWeek, eachDayOfInterval, eachWeekOfInterval, startOfMonth, endOfMonth } from 'date-fns';
+import { useState } from 'react';
+
+const Calendar: React.FC = () => {
+  const [targetDate, setTargetDate] = useState(new Date());
+  const [selectedDates, setSelectedDates] = useState<number[]>([]);
+
+  const getCalendarArray = (date: number | Date) => {
```

Comment:
getCalenderArrayにあたえるtargetDateはDate型で確定しているので、ここはDateのみの許容でいいかな

```suggestion
  const getCalendarArray = (date: Date) => {
```